src/samplers/cpu/mod.rs

```diff
@@ -150,6 +128,60 @@ impl Sampler for Cpu {
 }
 
 impl Cpu {
+    #[cfg(feature = "bpf")]
+    fn initialize_bpf_perf(&mut self) -> Result<(), std::io::Error> {
+        let cpus = crate::common::hardware_threads().unwrap();
+        let interval = self
+            .sampler_config()
+            .interval()
+            .unwrap_or_else(|| self.general_config().interval()) as u64;
```

Comment:
Maybe it is better to add `self.interval()` function instead. We already have a similar piece of code in [samplers/mod.rs](https://github.com/twitter/rezolus/blob/master/src/samplers/mod.rs#L65)

Reply:
Yes. You're right. Let me see about doing that in this PR.

Reply:
Fixed